app/src/main/java/weather/ekamp/com/weatherappkotlin/view/Landing.kt

```diff
@@ -50,13 +57,21 @@ class Landing : AppCompatActivity(), LandingView {
         weather_description.visibility = View.GONE
         temperature.visibility = View.GONE
         weather_representation.visibility = View.GONE
+        sync_button.animation?.let {
+            sync_button.animation.repeatCount = Animation.INFINITE
+        } ?:run {
+            var rotation = AnimationUtils.loadAnimation(this, R.animator.sync_rotator)
+            rotation.repeatCount = Animation.INFINITE
+            sync_button.startAnimation(rotation)
```

Comment:
We should make sure we stop the animation when the Activity dies, within onDestroy or onPause is a good place to check if we are sill animating, and if we are stopping the animation as the View is no longer visible